packages/sanity/src/core/form/studio/assetSource/AssetRow.tsx

```diff
@@ -42,12 +42,18 @@ const CustomCard = styled(Card)<RowProps>`
       --card-muted-fg-color: var(--card-bg-color);
       --card-fg-color: var(--card-bg-color);
     `}
+
+  ${({disabled}) =>
+    disabled &&
+    css`
+      --card-muted-fg-color: var(--card-border-color);
```

Comment:
make the text (beyond the title) greyed out

Reply:
> > We use the [attr-accept](https://www.npmjs.com/package/attr-accept) package to help us verify the mime type during drag+drop, so might serve us well to use it here too?
> 
> I see that we are using a `accepts` in [here](https://github.com/sanity-io/sanity/blob/next/packages/sanity/src/core/form/studio/uploads/accepts.ts) but it's not from that package but seems to have the same behavior and being used in the way you mentioned. 🙂 I'll explore a bit

Ah, yeah, that's the one! Looks like we have inlined it and added types :)